src/installer/managed/Microsoft.NET.HostModel/Bundle/FileEntry.cs

```diff
@@ -28,7 +28,7 @@ public class FileEntry
         public FileEntry(FileType fileType, string relativePath, long offset, long size)
         {
             Type = fileType;
-            RelativePath = relativePath.Replace(Path.DirectorySeparatorChar, DirectorySeparatorChar);
+            RelativePath = relativePath.Replace('\\', DirectorySeparatorChar);
```

Comment:
Won't this break if this is a unix path that actually has a `\` in the file name?

Reply:
It would, but I'm not sure what the experience should be for these cases... `dotnet new console -o 'a\b\c'` or having a C# file named `'x\y\z.cs'` and then doing `dotnet build` won't work either.

Reply:
You're right -- it looks like the rest of the tooling doesn't like that either. In that case, I'm fine with this fix. The scenario we'd be breaking really isn't supported today.